channeldb/circuit.go

```diff
@@ -0,0 +1,172 @@
+package channeldb
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/hex"
+	"github.com/boltdb/bolt"
+	"github.com/roasbeef/btcd/wire"
+)
+
+var (
+	// circuitBucket is the name of the bucket within the database that
+	// stores all data related to circuits.
+	circuitBucket = []byte("circuits")
```

Comment:
Minor nit: the `godoc` comment here should also describe how the keying schema _within_ the bucket is set up. As an example: what mappings are present in the top-level of this bucket? Or: are there deeper nested buckets within this one?